res/schema.xml

```diff
@@ -582,4 +582,31 @@ reapplying those migrations.
       UPDATE library SET filetype='aiff' WHERE filetype='aif';
     </sql>
   </revision>
+  <revision version="40" min_compatible="3">
+    <description>
+      Suggestion tables added.
+    </description>
+    <sql>
+      CREATE TABLE IF NOT EXISTS suggestion_library (
+        id INTEGER PRIMARY KEY AUTOINCREMENT,
+        artist varchar(48),
+        title varchar(48),
+        playcount INTEGER,
+        match FLOAT,
+        duration FLOAT,
+        location varchar(512));
+
+       CREATE TABLE IF NOT EXISTS suggestion_playlists (
+        id INTEGER PRIMARY KEY,
+        name varchar(100) UNIQUE
+        );
+
+      CREATE TABLE IF NOT EXISTS suggestion_playlist_tracks (
+        id INTEGER PRIMARY KEY AUTOINCREMENT,
+        playlist_id INTEGER REFERENCES suggestion_playlist(id),
+        track_id INTEGER REFERENCES suggestion_library(id),
+        position INTEGER);
```

Comment:
How is this calculated?

Reply:
This is actually order of the responses, I've added it due to setPlaylist method.